app/bundles/PluginBundle/Helper/IntegrationHelper.php

```diff
@@ -82,26 +75,32 @@ class IntegrationHelper
     private $byPlugin = [];
 
     /**
-     * IntegrationHelper constructor.
-     *
      * @param Kernel               $kernel
      * @param EntityManager        $em
      * @param PathsHelper          $pathsHelper
      * @param BundleHelper         $bundleHelper
      * @param CoreParametersHelper $coreParametersHelper
      * @param TemplatingHelper     $templatingHelper
      * @param PluginModel          $pluginModel
+     *
+     * @throws Exception
```

Comment:
Nitpick: please see above.
```suggestion
     * @throws \Exception
```